front/src/modules/timesStops/utils.ts

```diff
@@ -28,7 +28,13 @@ export const formatSuggestedViasToRowVias = (
   }
   return formattedOps.map((op, i) => {
     const pathStep = pathSteps.find((step) => matchPathStepAndOp(step, op));
-    const { arrival, onStopSignal, name, stopFor, theoreticalMargin } = pathStep || op;
+    const { arrival, onStopSignal, name, stopFor, theoreticalMargin } = pathStep || {
+      ...op,
+      stopFor: undefined,
+      arrival: undefined,
+      onStopSignal: false,
+      theoreticalMargin: undefined,
+    };
```

Comment:
Ah! If we don't want to grab some props from `op` then maybe we can split into two assignments?

```js
const { name } = pathStep || op;
const { arrival, onStopSignal, stopFor, theoreticalMargin } = pathStep || {};
```

Reply:
Done